### DISCUSSION TITLE
No RecordComponents in the parsed DOM node for RecordDeclaration

### DISCUSSION THREAD

**Discussion by gayanper** [eclipse-jdtls/eclipse-jdt-core-incubator#397] [Q&A]
Given the following code
```
package com.example;

public record Person(String name, int age) {
    
}
```

the found node looks like

<img width="747" alt="image" src="https://github.com/eclipse-jdtls/eclipse-jdt-core-incubator/assets/7471994/4629f985-000f-4fe4-a9b7-3ae18d18a825">

Any reason that we don't have the record components ?

**Comment by gayanper** [comment]
@mickaelistria is this something still not implemented ?

**Reply by mickaelistria** [reply]
Yes, it seems to be missing in JavacConverter, the recordComponents are never filled.

**Reply by gayanper** [reply]
Is someone working on this adding record components ?

**Reply by gayanper** [reply]
PR https://github.com/eclipse-jdtls/eclipse-jdt-core-incubator/pull/399
